software/wmla120.py

```diff
@@ -0,0 +1,1871 @@
+#! /usr/bin/env python
+# Copyright 2018 IBM Corp.
+#
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import nested_scopes, generators, division, absolute_import, \
+    with_statement, print_function, unicode_literals
+
+import argparse
+import glob
+import os
+import platform
+import re
+import sys
+from shutil import copy2
+import calendar
+import time
+import yaml
+import json
+from getpass import getpass
+import pwd
+import grp
+import click
+
+import lib.logger as logger
+from repos import PowerupRepo, PowerupRepoFromDir, PowerupYumRepoFromRepo, \
+    PowerupAnaRepoFromRepo, PowerupRepoFromRpm, setup_source_file, \
+    PowerupPypiRepoFromRepo, get_name_dir
+from software_hosts import get_ansible_inventory, validate_software_inventory
+from lib.utilities import sub_proc_display, sub_proc_exec, heading1, Color, \
+    get_selection, get_yesno, rlinput, bold, ansible_pprint, replace_regex
+from lib.genesis import GEN_SOFTWARE_PATH, get_ansible_playbook_path
+
+from engr_mode import pre_post_file_collect, dependency_folder_collector
```

Comment:
The 'engr_mode.py' module was removed by 94ad9bb (I would have missed this if not for unit test - thanks @jjalvare!)

Reply:
yw!

Reply:
resolved with commit b716edc